tests/e2e/gc_block_volume_expansion.go

```diff
@@ -1628,6 +1640,293 @@ var _ = ginkgo.Describe("[csi-guest] Volume Expansion Test", func() {
 
 	})
 
+	/*
+	   This test verifies the static provisioning workflow in guest cluster when svcPVC=gcPVC.
+
+	   Test Steps:
+	   1. Create FCD with valid storage policy on gc-svc.
+	   2. Create Resource quota
+	   3. Create CNS register volume with above created FCD on SVC
+	   4. verify svc-PV, svc-PVC got created , check the bidirectional reference on svc.
+	   5. On GC create a gc-PV by pointing volume handle got created by static provisioning on svc-PVC (in step 4)
+	   6. On GC create a gc-PVC pointing to above created PV (step 5)
+	   7. Wait for gc-PV , gc-PVC to get bound
+	   8. Create POD, verify the status
+	   9. Trigger online volume expansion on gc-pvc and make sure volume expansion is successful.
+	   9. Delete all the above created PV, PVC and resource quota
+	*/
+	ginkgo.It("[csi-guest] Online volume resize on statically created PVC on guest cluster svcPVC=gcPVC", func() {
```

Comment:
`[csi-guest]` is already part of `Describe()`, not required here again for individual tests.